server/api/index.js

```diff
@@ -1,8 +1,10 @@
 const express = require('express');
 const users = require('./routes/users');
+const auth = require('./routes/auth');
 
 const router = express.Router();
 
-router.use('/users', users);
+router.use('/api', users);
```

Comment:
per our convo on discord

```suggestion
router.use('/api/users', users);
```